tests/test_patchqueue.py

```diff
@@ -27,11 +27,11 @@ def tearDown(self):
         shutil.rmtree(self.test_dir)
 
     def test_patch_guards(self):
+        """Patch guards are applied correctly"""
         series = ("patch_no_guard_or_comment\n",
                   "patch_with_a_comment # this is a comment\n",
                   "patch_with_a_positive_guard #+aguard\n",
-                  "patch_with_a_negative_guard #-aguard\n",
-                  )
```

Comment:
If you leave the parenthesis on a new line, adding a new patch will not require two line changes.